server/controllers/cv-controller.js

```diff
@@ -0,0 +1,44 @@
+const client = require('../lib/github-client');
+
+console.log(client.getUserProfile());
+function fetchingData(username, res) {
+	const userProfile = client.getUserProfile(username);
+	const PullRequest = client.getUserPullRequests(username);
+	return Promise.all([userProfile, PullRequest])
+  .then((data
+  	) =>{
+  	console.log(data);
+    return res.render('index', {code:data[0], codes:data[1][0]});
+  }).catch(err => {
+    console.log('Not retriving Data');
```

Comment:
Remember when you console log it only shows up in the node console - this error message won't get sent to the user. You could use res.send, or even res.render to send an error message to the user, rather than having their request just hang